fw/preamp/src/syscalls.c

```diff
@@ -0,0 +1,204 @@
+/**
```

Comment:
Is this file needed? Try removing it

Reply:
I missed this one! Compiling without it threw no errors, and the binary didn't change at all